app/helpers/application_helper.rb

```diff
@@ -15,7 +15,7 @@ def generate_history_text url
       puts "#"*20
       puts e.message
       puts "#"*20
-      return url
+      return "/" # default to home page.
```

Comment:
Does it make more sense to default to homepage or just the entire URL?  The reason I had for URL was that just because it can't be parsed doesn't make it invalid.  This would send them back to the page that got them in to Fulfillment. Not sure what the desired functionality should be but I will check with our stakeholders for sure.